docs/content/concepts/placement-locations-and-scheduling.md

```diff
@@ -170,12 +173,14 @@ workspace.
 ### Resource Upsyncing
 
 In most cases kcp will be the source for syncing resources to the `SyncTarget`, however, in some cases,
-kcp would need to receive a resource that was provisioned by a controller on the `SyncTarget`.
-This is the case with storage PVs, which are created on the `SyncTarget` by a CSI driver.
+kcp would need to receive a resource that was provisioned by a controller on the pcluster.
+This is the case with storage PVs, which are created on the pcluster by a CSI driver.
 
 Unlike the `Sync` state, the `Upsync` state is exclusive, and only a single `SyncTarget` can be the source of truth for an upsynced resource.
 In addition, other `SyncTargets` cannot be syncing down while the resource is being upsynced.
 
+Upsyncing is initiated by something creating a label with key = `state.workload.kcp.io/<sync-target-key>` and value = `UpSync` on the object in the pcluster.
+
```

Comment:
The value is "Upsync" (with lowercase "s") as can be seen in https://github.com/kcp-dev/kcp/blob/dc49b99881374f748949250a8ff3b737f019ef0c/pkg/apis/workload/v1alpha1/types.go#L34

Reply:
Thanks. Fixed in the force-push to d5b1b8c3